projects/packages/forms/src/blocks/contact-form/child-blocks.js

```diff
@@ -430,7 +435,7 @@ export const childBlocks = [
 				...FieldDefaults.attributes,
 				label: {
 					type: 'string',
-					default: 'URL',
+					default: 'Website',
```

Comment:
Needs to be wrapped in `__()`?

Reply:
TBH I'm not sure about the process in this repo. Probably yes, so I updated it now. I haven't encountered something similar while working on GB because I think there is an extraction mechanism for block.json strings. Here is a `JS` file so it seems sensible to do it.

Noting though that the previous value wasn't using `__` function and in that file there are many inconsistencies when it's used. I think an issue for handling that to be consistent should be created.

Reply:
Yeah, `block.json` gets extracted like in Gutenberg but the  `.js/jsx/ts/tsx` files need `__()` function calls. I'm sure it was just an oversight not to have it here.